System/RTETools.h

```diff
@@ -38,36 +42,36 @@ namespace RTE {
 
 #pragma region Random Numbers
 	/// <summary>
-	/// Seeds the rand with the current runtime time.
+	/// Seed the mt19937 random number generator.
+	/// </summary>
+	void SeedRand(unsigned int seed);
+
+	/// <summary>
+	/// Uniformly distributed random double in the range [min, max].
+	/// The min must be lesser or equal to max.
 	/// </summary>
-	void SeedRand();
+	double UDRand(double min, double max);
```

Comment:
Resolving this in favor of: [comment](https://github.com/cortex-command-community/Cortex-Command-Community-Project-Source/pull/145#discussion_r479582774).
Also there's been discussion in discord since this comment was made.
Main outcome is usage of function templates which eliminate PosRand, SelectRand and RangeRand in favor of RandomNum.